segmentation_processing/README.md

```diff
@@ -0,0 +1,45 @@
+# STEGO Staging
+This directory holds files related to working with the STEGO model provided by ASU. STEGO model implementation by Blake Harrison. Server script modifications by Jacob Lichtefeld.
+
+This server runs in a Python 3.8 environment.
+
+## Creating the Segmentation Environment Locally
+
+To create a local environment for this directory either make a conda environment or a python venv then:
+
+```bash
+# activate your environment
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+```
+
+## Cluster Configuration
+
+With `segmentation_processing` as the working directory, run the following command, which will create a virtual environment (`venv`) with the necessary dependencies in the working directory, as well as download the STEGO model.
+
+```bash
+PATH="/nas/gaia/adam/shared/local_python/bin/:$PATH" make install
+```
+
+### Creating and upgrading a local Python instance
+
+The easiest way we've found is to create a local Python instance is with Conda, which also has the advantage of using a pre-compiled `python` executable that has no linking issues.
+
+The following commands were used to create a recent local Python:
+
+```bash
+pyenv install miniconda3-3.8-4.12.0
+pyenv local miniconda3-3.8-4.12.0
+conda create --channel conda-forge --prefix ./local_python python=3.8.13
+pyenv local --unset
+```
+
+When updating to a new minor version of Python, the original `venv` needs to be deleted first or else no new `venv` will be created. The `venv` cannot be upgraded easily in-place because most non-pure Python packages must be reinstalled to work on a different minor version.
```

Comment:
I think everything from this line down is "how to keep the Python env up-to-date" and therefore should have it's own header. Correct?

Reply:
This is how to update the base python version within the `venv` not just the requirements of the venv. It's specific to the creation process this header already has.